app/mailers/reservation_mailer.rb

```diff
@@ -6,6 +6,14 @@ def reservation_requested
     mail(to: @reservation.submitted_by.email, subject: @subject)
   end
 
+  def review_requested
+    @reservation = params[:reservation]
+    @library = @reservation.library
+    @admins = User.where(library_id: @library.id).where(role: ["admin", "super_admin"]).pluck(:email)
```

Comment:
```suggestion
    @admins = User.where(library_id: @library.id).where(role: ["admin"]).pluck(:email)
```

I think we don't want to send emails to `super_admin` users. That wasn't super well specified in the task, but super admins are the folks who (in theory) would edit which libraries are in the system. It's a mostly unused role since we are only supporting CTL currently.

Reply:
Ah okay that makes sense! Thanks for the context! Change has been made :)